easybuild/easyblocks/p/psmpi.py

```diff
@@ -58,9 +124,59 @@ def extra_options(extra_vars=None):
             'mpich_opts': [None, "Optional options to configure MPICH", CUSTOM],
             'threaded': [False, "Enable multithreaded build (which is slower)", CUSTOM],
             'pscom_allin_path': [None, "Enable pscom integration by giving its source path", CUSTOM],
+            'pgo': [False, "Enable profiling guided optimizations", CUSTOM],
+            'mpiexec_cmd': ['srun -n ', "Command to run benchmarks to generate PGO profile. With -n switch", CUSTOM],
```

Comment:
mpirun is a better default, that usually works even outside the batch system.

Reply:
`mpiexec -np` is the only one that is actually part of the standard, if you were to go for a generic default then it should really be that